cpp/ql/src/experimental/Security/CVE/cve-2016-6480.ql

```diff
@@ -0,0 +1,34 @@
+import cpp
```

Comment:
This query is missing all the required meta-data information. You can read about the required meta-data in `2. Query metadata` under the `Submitting a new experimental query
` scion in our [CONTRIBUTING.md](https://github.com/github/codeql/blob/main/CONTRIBUTING.md#submitting-a-new-experimental-query) file.

In particular, you need to add:
1. A unique `@id` tag
2. A `@name` tag
3. A `@description` tag
4. A `@kind` tag. Note that, depending on the `@kind` you choose, you'll have to adjust your `select` clause appropriately. You can read about the required columns: https://codeql.github.com/docs/writing-codeql-queries/about-codeql-queries/#select-clause.
5. A `@problem.severity` tag.